### PR TITLE
FORMIT-10058: Update DynamoText: add ability to specify font and tolerance, organize letter curves into sublists

### PR DESCRIPTION
**Pull request by josh-goldstein-adsk**

### CODE REVIEW

**Comment by josh-goldstein-adsk** [comment]
@mjkkirschner @reddyashish FYI. I can't seem to add you as reviewers, but can you take a look?

File: DynamoText.csproj [hunk 1]
```diff
@@ -32,20 +33,18 @@
   <ItemGroup>
     <Reference Include="DynamoServices, Version=2.0.0.4604, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>packages\DynamoVisualProgramming.DynamoServices.2.0.0\lib\net45\DynamoServices.dll</HintPath>
-      <Private>False</Private>
```
Comment (mjkkirschner) [inline DynamoText.csproj:35]: please mark these true so that the files are not copied to the bin folder

File: DynamoText.csproj [hunk 2]
```diff
@@ -32,20 +33,18 @@
...
     </Reference>
     <Reference Include="DynamoUnits, Version=2.0.0.4604, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>packages\DynamoVisualProgramming.ZeroTouchLibrary.2.0.0\lib\net45\DynamoUnits.dll</HintPath>
-      <Private>False</Private>
```
Comment (mjkkirschner) [inline DynamoText.csproj:39]: please mark these true so that the files are not copied to the bin folder

File: DynamoText.csproj [hunk 3]
```diff
@@ -32,20 +33,18 @@
...
     </Reference>
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
     <Reference Include="ProtoGeometry, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>packages\DynamoVisualProgramming.ZeroTouchLibrary.2.0.0\lib\net45\ProtoGeometry.dll</HintPath>
-      <Private>False</Private>
```
Comment (mjkkirschner) [inline DynamoText.csproj:45]: please mark these true so that the files are not copied to the bin folder

File: Text.cs [hunk 4]
```diff
@@ -71,6 +74,76 @@ public static IEnumerable<Curve> FromStringOriginAndScale(string text, Point ori
             return crvs;
         }
 
+        public static IEnumerable<IEnumerable<Curve>> ByStringFontToleranceOriginScale(string text, string fontName, float tolerance, Point origin, double scale)
```
Comment (mjkkirschner) [inline Text.cs:77]: please add /// summary so that we get description in dynamo

**Pull request closed without merging**